user-guide-bianca.md

```diff
@@ -0,0 +1,71 @@
+
+## Pull the images with singularity
+
+### Pull images
+* Login to Rackham running:
+```bash
+ssh <username>@rackham.uppmax.uu.se
+```
+Navigate to the folder you want to pull the images.
+* To download the crypt4gh key generator image run the following command in the terminal:
+```bash
+singularity pull docker://nbisweden/ega-crypt4gh:crypt4gh-keygen-1.0.0
+```
+* To download the crypt4gh image run the following command in the terminal:
+```bash
+singularity pull docker://nbisweden/ega-crypt4gh:crypt4gh-1.0.0
+```
+Now you should be able to see two files with `.sif` extensions in the folder when running the `ls` command.
+
+### Move the singularity images to Bianca
+You can move the singularity images to Bianca using wharf. The guide for transferring files from and to Bianca can be found [here](https://www.uppmax.uu.se/support/user-guides/transit-user-guide/).
+
+In summary, the steps needed are the following:
+In a terminal window
+* Login to transit running:
+```bash
+ssh <username>@transit.uppmax.uu.se
+```
+* Mount to the bianca project folder using the following command
+```bash 
+mount_wharf <project_name>
+```
+where [project_name] in the form of sens20XXX.
+* Copy image files using the cp command and replacing to the correct locations:
+```bash 
+cp ega-crypt4gh_crypt4gh-1.0.0.sif <project_name>/
+cp ega-crypt4gh_crypt4gh-keygen-1.0.0.sif <project_name>/
+```
+
+In another terminal
+* Login to bianca running:
+```bash
+ssh -A <username>-<project_name>@bianca.uppmax.uu.se
+```
+* Copy image files from `/proj/<project_name>/nobackup/wharf/<username>/<username>-<project_name>` to the folder you want to store the images using the cp command and replacing to the correct locations:
+```bash 
+cp cp /proj/<project_name>/nobackup/wharf/<username>/<username>-<project_name>/ega-crypt4gh_crypt4gh-1.0.0.sif .
+cp cp /proj/<project_name>/nobackup/wharf/<username>/<username>-<project_name>/ega-crypt4gh_crypt4gh-keygen-1.0.0.sif .
+```
+Running the `ls` command in this folder, you should be able to see the two `.sif` files.
+
+## Key pair generation
+In order to encrypt the file(s) with crypt4gh, you need both a personal key pair and the EGA-SE public encryption key.
+### Create a personal key pair
+The creation of a key pair is very simple using crypt4gh. Run the following command, in the folder containing the `.sif` files, replacing [my-key] with the name of the key (can be anything) and specifying a passphrase when requested:
+```bash
+singularity run ega-crypt4gh_crypt4gh-keygen-1.0.0.sif --pk <my-key>.pub --sk <my-key>.sec
+```
```

Comment:
I do not have complete overlook here, is there any benefit for them of keeping track of the crypt4gh key (as in "keys are registered to an account and output files will only be encrypted to that key").

From what I know, they need a key to be able to use crypt4gh, but there is no real need for them to save it (and when I try on my system now, it seems to behave as I remembered it should (running fine without a precreated key).

Reply:
I think you are right on this one. I am not sure if there is a reason for them to keep track of the key. Maybe @viklund has a saying